sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -150,6 +183,7 @@ class JDBCOptions(
 }
 
 object JDBCOptions {
```

Comment:
Move `JDBCOptions` in the end of this file

Reply:
@maropu Sure.